### PR TITLE
Added Mission Kontrol

### PR DESCRIPTION
**Pull request by duncan-m**
Added self hosted admin panel MissionKontrol to Misc/Other

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [ ] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [ ] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [ ] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [ ] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software)
  must be added to `non-free.md` and marked `⊘ Proprietary`:
  ``- [Name](http://homepage/) `⊘ Proprietary` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``
- [ ] Additions are inserted preserving alphabetical order.
- [ ] Additions are not already listed at any of
  - [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management),
  - [awesome-analytics](https://github.com/onurakpolat/awesome-analytics) (analytics),
  - [staticgen.com](https://www.staticgen.com/)
  - [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [ ] The `Language` tag is the main server-side requirement for the software - don't include frameworks or specific dialects.
- [ ] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [ ] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [ ] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [ ] Any software project you are adding to the list is actively maintained.
- [ ] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

**Comment by nodiscc** [comment]
I'd rather have it added to https://github.com/n1trux/awesome-sysadmin#control-panels but since awesome-sysadmin was archived... I need to think a bit about it (do we direct sysadmin-related additions to https://awesome.tilde.fun/t/sysadmin?  Do we merge back all control panels and other sections we delegated to awesome-sysadmin? ...?)

**Comment by duncan-m** [comment]
It's an interesting one. Admin panels aren't usually set-up or driven by sysadmins. Usually they're developer, founder or product manager led. Sysadmins in my experience prefer the command line and/or legacy tools.

It's definitely a growing category. 5 years ago there were only plugins/language extensions like railsadmin now there's a whole bunch of fully independent tools like MissionKontrol that are definitely an evolution forwards from things like PHPmyadmin/complete self-code. 

Use cases vary too, could be anything from someone wanting to edit data in a database of weather data they maintain to master data management to tech company doing customer support.

**Comment by nodiscc** [comment]
> Admin panels aren't usually set-up or driven by sysadmins.
> 5 years ago there were only plugins/language extensions 

Not really. https://github.com/rundeck/rundeck and custom CI/Jenkins dashboards have existed for a while for operations. There are countless BI tools that run predefined queries against databases. etc.

But I see your point, Mission Kontrol is not directed towards sysadmins, it makes sense to have it here.

**Review by nodiscc — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1046,6 +1046,7 @@ _See also <https://en.wikipedia.org/wiki/List_of_streaming_media_systems>, <http
 - [MailyGo](https://codeberg.org/jlelse/MailyGo) - MailyGo is a small tool written in Go that allows to send HTML forms, for example from static websites without a dynamic backend, via email. `MIT` `Go`
 - [MalwareMultiScan](https://github.com/mindcollapse/MalwareMultiScan) - Self-hosted VirusTotal-like for scanning files with multiple antiviruses running in a Docker. ([Demo](http://199.247.24.56:8888/)) `MIT` `C#`
 - [MindsDB](https://github.com/mindsdb/mindsdb) - MindsDB is an open source self hosted AI layer for existing databases that allows you to effortlessly develop, train and deploy state-of-the-art machine learning models using standard queries. `GPL-3.0` `Python`
+- [MissionKontrol](https://www.missionkontrol.io) - Configurable admin panel allowing non-technical users to CRUD data on MySQL/PostGRES databases. ([Source Code](https://github.com/Mission-Kontrol/MissionKontrol-rails)) `AGPL-3.0` `Ruby`
```
Comment (n8225) [inline README.md:1049]: https://github.com/Mission-Kontrol/MissionKontrol-rails lists license as GPL-3.0

**Review by n8225 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nodiscc) [reply README.md:1049]: So does https://github.com/Mission-Kontrol/MissionKontrol/blob/develop/LICENSE. @duncan-m can you update the license?

**Comment by duncan-m** [comment]
Sorry I saw the request. Will get it updated hopefully today. I think the listing here is correct it's the repo that's been published with the wrong license.

**Comment by duncan-m** [comment]
This is now updated on the MK repo

**Review by nodiscc — APPROVED** [review]

**Pull request merged**